ansible_collections/arista/avd/roles/eos_cli_config_gen/docs/tables/ethernet-interfaces.md

```diff
@@ -1156,6 +1199,96 @@
                 #   - 1-10
               - range: <str; required; unique>
                 mac_address_maximum: <int>
+          tap:
+
+            # VLAN ID or range of VLAN IDs within range 1-4094.
+            allowed_vlan: <str>
+            default:
+
+              # Tap group names for the interface.
+              groups:
+                - <str>
+
+              # Interfaces like -  Ethernet, InternalRecirc, Port-Channel, Recirc-Channel.
+              interfaces:
+                - <str>
+
+              # Default nexthop-group names.
+              nexthop_groups:
+                - <str>
+            identity:
+
+              # Tap port ID tag.
+              port_id: <int; 1-4094>
+
+              # Inner tap port ID tag.
+              inner_port_id: <int; 1-4094>
+
+            # Pop all MPLS labels.
+            mpls_pop_all: <bool>
+
+            # Native VLAN ID when interface is in tap mode.
+            native_vlan: <int; 1-4094>
+            truncation:
+              enabled: <bool; required>
+
+              # Ingress packet truncation size in bytes.
+              size: <int; 100-9236>
+            mac_address:
+
+              # MAC address for the source.
+              src: <str>
+
+              # MAC address for the destination.
+              dest: <str; required>
+            encapsulation:
+
+              # Strip VXLAN encapsulation header.
+              vxlan_strip: <bool>
+
+              # Strip GRE encapsulation header.
+              gre_strip: <bool; required>
+              gre:
+
+                # Destination IP address of tunnel packets.
+                destination: <str>
+
+                # Source IP address of tunnel packets.
+                source: <str>
+
+                # Protocol type in GRE header.
+                # Protocol range: 0x0000-0xFFFF
+                protocol: <str>
+          tool:
```

Comment:
One more option
```
wan1-dc1(config-if-Et1)#switchport tool dot1q remove ?
  outer  Remove outer tag

wan1-dc1(config-if-Et1)#switchport tool dot1q remove ?
  outer  Remove outer tag

wan1-dc1(config-if-Et1)#switchport tool dot1q remove outer ?
  $      list end
  <1-2>  Specify indices of vlan tags to be removed
```

Reply:
Also we need to make this as list because we can have multiple command with different source and destination address

Reply:
Added dot1q schema and changes the gre_strip to a list